From 2c6e598a30480c17308c79710cca954732560060 Mon Sep 17 00:00:00 2001
From: Alain Volmat <alain.volmat@foss.st.com>
Date: Thu, 13 Oct 2022 18:44:08 +0200
Subject: [PATCH 0116/1141] arm64: dts: st: enable the i2c2 node in
 stm32mp257f-ev.dts

Enable the i2c2 node for the STM32MP257F eval board.

Signed-off-by: Alain Volmat <alain.volmat@foss.st.com>
Change-Id: I8c1cb756ea46a49d0b0215de658c5909a78e7926
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/302061
Tested-by: Valentin CARON <valentin.caron@foss.st.com>
Reviewed-by: Valentin CARON <valentin.caron@foss.st.com>
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev.dts | 13 +++++++++++++
 1 file changed, 13 insertions(+)

--- a/arch/arm64/boot/dts/st/stm32mp257f-ev.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev.dts
@@ -37,6 +37,19 @@
 	status = "okay";
 };
 
+&i2c2 {
+	pinctrl-names = "default", "sleep";
+	pinctrl-0 = <&i2c2_pins_a>;
+	pinctrl-1 = <&i2c2_sleep_pins_a>;
+	i2c-scl-rising-time-ns = <185>;
+	i2c-scl-falling-time-ns = <20>;
+	clock-frequency = <100000>;
+	status = "okay";
+	/* spare dmas for other usage */
+	/delete-property/dmas;
+	/delete-property/dma-names;
+};
+
 &rtc {
 	status = "okay";
 };
